opentimelineio/schema/gap.py

```diff
@@ -32,6 +35,36 @@ class Gap(core.Item):
     _serializable_label = "Gap.1"
     _class_path = "schema.Gap"
 
+    def __init__(
+        self,
+        name=None,
+        # note - only one of the following is accepted
```

Comment:
Super super minor, but perhaps add "two" ("only one of the following two.."). At first read I thought this applied to effects/markers/etc as well.

Reply:
I agree.